properties/light.py

```diff
@@ -26,8 +26,9 @@
 )
 
 NORMALIZEBYCOLOR_DESCRIPTION = (
-    "Normalize intensity by the Color Luminance; Radiometric units like Power should not be "
-    "normalized, while Photometric units like Lumen, Candela, Lux should"
+    "Normalize intensity by the Color Luminance.\n"
+    "Recommended for Photometric units (Lumen, Candela, Lux) to simulate \n"
+    "the luminous efficiency function."
```

Comment:
Remove the dot at the end of the description, otherwise Blender will complain and there will be two dots in the tooltip.